tests/integration/targets/provision_virtual_esxi_test/runme.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+source ../init.sh
+ansible-playbook mock_side_effects.yml  &
```

Comment:
from @machacekondra 

> What about having an async task doing the side effect, instead of another ansible-playbook execution?
https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_async.html

I tried this initially but couldnt get it to work. I would prefer the async approach though if I missed something. 
The issue is that I need this side effect task/playbook to loop until it succeeds or the other playbook ends. Async tasks will only run once and cannot be retried

Reply:
Ah, right.